utils/cli/src/table.rs

```diff
@@ -0,0 +1,188 @@
+
+    table.set_titles(Row::new(
+        columns
+            .iter()
+            .map(|c| {
+                Cell::new(c)
+                    .with_style(Attr::Bold)
+                    .with_style(Attr::ForegroundColor(color::GREEN))
+            })
+            .collect(),
+    ));
+    if values.is_empty() {
+        let _ = table.add_row(columns.iter().map(|_| Cell::new("")).collect());
+    }
+    for row in values {
+        if let Some(row_items) = row.as_array() {
+            use serde_json::Value;
+
+            let row_strings = row_items
+                .iter()
+                .map(|v| match v {
+                    Value::String(s) => s.to_string(),
+                    Value::Null => "".into(),
+                    v => v.to_string(),
+                })
+                .collect();
+            table.add_row(row_strings);
+        }
+    }
+    if !summary.is_empty() {
+        table.add_row(Row::empty());
+        table.add_empty_row();
+        let l = summary.len();
+        for (idx, row) in summary.into_iter().enumerate() {
+            if let Some(row_items) = row.as_array() {
+                use serde_json::Value;
+
+                let row_strings = Row::new(
+                    row_items
+                        .iter()
+                        .map(|v| {
+                            let c = Cell::new(&match v {
+                                Value::String(s) => s.to_string(),
+                                Value::Null => "".into(),
+                                v => v.to_string(),
+                            });
+
+                            if idx == l - 1 {
+                                c.with_style(Attr::Bold)
+                            } else {
+                                c
+                            }
+                        })
+                        .collect(),
+                );
+                table.add_row(row_strings);
+            }
+        }
+    }
+    let _ = table.printstd();
+}
+
+pub fn print_json_table(
+    columns: &Vec<String>,
+    values: &Vec<serde_json::Value>,
+) {
+    let columns_size_eq_values_size = values.iter().all(|row| match row {
+        serde_json::Value::Array(values) => values.len() == columns.len(),
+        _ => false,
+    });
+    if columns_size_eq_values_size {
+        let kvs: Vec<HashMap<&String, &serde_json::Value>> = values
+            .iter()
+            .map(|row| match row {
+                serde_json::Value::Array(row_values)
+                    if columns.len() == row_values.len() =>
+                {
+                    columns
+                        .into_iter()
+                        .enumerate()
+                        .map(|(idx, key)| (key, &row_values[idx]))
+                        .collect()
+                }
+                _ => unreachable!(),
+            })
+            .collect();
+        println!(
+            "{}",
+            serde_json::to_string_pretty(&serde_json::json!(kvs)).unwrap()
```

Comment:
Consider removing unwraps from this module and return Result<()>

Reply:
Done.

Reply:
Done.